internal/providers/cloudstack/cloudstack.go

```diff
@@ -202,7 +201,12 @@ func fetchConfigFromDevice(logger *log.Logger, ctx context.Context, label string
 	}
 	defer func() {
 		_ = logger.LogOp(
-			func() error { return unix.Unmount(mnt, 0) },
+			func() error {
+				if err := util.UmountPath(mnt); err != nil {
+					return err
+				}
+				return nil
```

Comment:
Isn't this just `return util.UmountPath(mnt)`?

Reply:
Yeah, makes sense!